control/fixtures/fixtures.go

```diff
@@ -155,3 +156,20 @@ func (m MockRequestedMetric) Version() int {
 func (m MockRequestedMetric) Namespace() core.Namespace {
 	return m.namespace
 }
+
+func NewExecutablePlugin(a plugin.Arg, path string) (*plugin.ExecutablePlugin, error) {
+	// Travis optimization: Try starting the plugin three times before finally
+	// returning an error
```

Comment:
Thanks, that was a copy paste issue.
As a matter of fact, this function turned out to be unneeded.
fixed [here](https://github.com/intelsdi-x/snap/pull/1591/files/5e69931d5eb7c0449ac83cd4d00ea4d0a7c191d6..9365564f29304f562cd73499a34fe7178162402f#diff-6af5c0382af05da36e9e39f37b2c4345R159)